### ISSUE TITLE
One account can upload, the other cannot.

### ISSUE THREAD

**Issue by stevennt** [makiisthenes/TiktokAutoUploader#88]
There is something with the region being assigned to an account by Tiktok that this app doesn't handle yet. I tried with couple of accounts. The app can upload to one account properly and reliably. It cannot upload to another account even when cli could get all the session details (even causing an account to be locked).

**Comment by makiisthenes** [comment]
Yes, currently region is hard coded, will need to set this dynamically.
